packages/beacon-node/src/api/impl/validator/index.ts

```diff
@@ -512,10 +512,10 @@ export function getValidatorApi({
 
       const fork = config.getForkName(slot);
       // set some sensible opts
-      // builderSelection will be deprecated and will run in mode Default if builder is enabled
+      // builderSelection will be deprecated and will run in mode MaxProfit if builder is enabled
       // and the actual selection will be determined using builderBoostFactor passed by the validator
-      builderSelection = builderSelection ?? routes.validator.BuilderSelection.Default;
-      builderBoostFactor = builderBoostFactor ?? BigInt(90);
+      builderSelection = builderSelection ?? routes.validator.BuilderSelection.MaxProfit;
+      builderBoostFactor = builderBoostFactor ?? BigInt(100);
```

Comment:
I reverting the changes on the beacon node side as in case of Lodestar the boost factor is dictated by the validator client and setting a different default here is undocumented behavior we should avoid.